tests/ci/spec_to_test_mapping.py

```diff
@@ -42,6 +42,10 @@
         'scotch',
         'zlib-devel'
     ],
+    '../components/parallel-libs/fftw/SPECS/fftw.spec': [
+        'fftw',
+        'openmpi4-gnu12-ohpc mpich-gnu12-ohpc'
+    ],
```

Comment:
Should I add entries also for all tests which `rm_execution` has been modified below ?

Reply:
I'll do it!